plugins/dell-dock/fu-dell-dock-i2c-ec.c

```diff
@@ -781,9 +780,23 @@ fu_dell_dock_ec_write_fw (FuDevice *device,
 	if (fw == NULL)
 		return FALSE;
 	data = g_bytes_get_data (fw, &fw_size);
-	write_size =  (fw_size / HIDI2C_MAX_WRITE) >= 1 ? HIDI2C_MAX_WRITE : fw_size;
-
 	dynamic_version = g_strndup ((gchar *) data + self->blob_version_offset, 11);
+
+	/* meet the minimum EC version */
+	if ((flags & FWUPD_INSTALL_FLAG_FORCE) == 0) {
+		if (self->data->module_type != MODULE_TYPE_130_USB4) {
+			if (fu_common_vercmp_full (dynamic_version, self->ec_minimum_version,
+				   FWUPD_VERSION_FORMAT_QUAD) < 0) {
```

Comment:
new logic looks good, but can you align the text better?  project has it doing things like this instead:

```
g_set_error (error, 
	     FWUPD_ERROR, 
	     FWUPD_ERROR_NOT_SUPPORTED,
	     "New EC version %s is less than minimum required %s",
	     dynamic_version, self->ec_minimum_version);
```
when it's a long one.